src/DatePicker/DatePicker.js

```diff
@@ -333,6 +333,7 @@ class DatePicker extends Component {
             localizedText,
             onBlur,
             onDatePickerClose,
+            openToDate,
             popoverProps,
```

Comment:
Should we add `openToDate` to `DatePicker.propTypes` too?

Reply:
I believe that is covered by `...Calendar.PropTypes`

Reply:
Right.

Wouldn't the de-structuring then be
```js
const { calendarProps: openToDate } = this.props
```
and story be
```js
<DatePicker calendarProps={{openToDate: _openToDate}} />;
```
?

Reply:
I think there is just a discrepancy for how we handle some Calendar props. Some of them are raw props to spread to the Calendar's HTML elements:
`calendarProps: PropTypes.shape({
        monthListProps: PropTypes.object,
        tableBodyProps: PropTypes.object,
        tableHeaderProps: PropTypes.object,
        tableProps: PropTypes.object,
        yearListProps: PropTypes.object
    })`

While the others that specify logic for dates and such (`blockedDates`, `disabledDates`, etc.) are pulled up a level to make for a simpler API and not having a huge `calendarProps` object.

Seems to be out of scope for this PR, though.